.env.tpl

```diff
@@ -19,6 +19,7 @@ AUTH_SECRET=keycloakZaakafhandelcomponentClientSecret
 AUTH_SERVER=http://localhost:8081
 BAG_API_CLIENT_MP_REST_URL=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_CLIENT_MP_REST_URL
 BAG_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_KEY
+BRON_ORGANISATIE=123443210
```

Comment:
Are these two org numbers actual real life organisations btw? Are there org numbers that are specifically meant for testing purposes? If so, would be good to use them.

Reply:
Bron is a dummy BSN. Will check for verantwoordelijke